_submissions/lua_script/Announcekill.md

```diff
@@ -0,0 +1,15 @@
+Script info
+
+Name: AnnunceKill
+Author: Roberty
+Repository: https://github.com/merdlin/ElunasScripts/blob/master/killanuncio.lua
+Download: https://github.com/merdlin/ElunasScripts/archive/master.zip
+License: AGPL
+Script integration
+Includes configuration (.ext)?: No, configuration is in the script.
+Includes SQL patches?: No.
+Description
+the script announces the death of a player in a battle
```

Comment:
What do you mean? it announces to every other players on the server?

Reply:
yes